pandunia-loge.csv

```diff
@@ -3374,6 +3375,7 @@ tiran:ia|tirania||||tyranny||tiranía||||||||||||||tyrannia|tyrania
 Titan:|Titan|planete 6 lune 6|||Titan|||||||||||||||||
 Titan:ia|Titania|planete 7 lune 4|||Titania|||||||||||||||||
 titan.em:|titanem|mate|Ti||titanem||titanio|||||||||||||titanio|titaani|tytan
+toal:|toal|||fra:touaille, eng:towel, spa:toalla, por:toalha, hin:तौलिया (tauliyā), ben:তোয়ালে (toyale), tgl:tuwalya, jpn:タオル (taoru)|towel||toalla|toalha|||||||तौलिया|||||viŝilo|pyyhe|ręcznik
```

Comment:
Hmm, "oaze" and "toal" are the first words with oa in Pandunia. I would like to rearrange them so that towel or cloth is "tual". It would be more like French "toile" /twal/ and its meaning can be broadened too.

The word for oasis, which really is etymologically related to Arabic "wāḥa", is actually more international than "vase", so that should be the first word. Perhaps vase can be "vas"...